binlog_streamer.go

```diff
@@ -128,7 +128,6 @@ func (s *BinlogStreamer) Run() {
 
 		if err != nil {
 			s.ErrorHandler.Fatal("binlog_streamer", err)
-			return
```

Comment:
Why remove these returns? I know we'll panic anyway but it seems more correct to return anyway

Reply:
In the beginning, ErrorHandler was a goroutine and Fatal send the error down a channel. Since be4f15da531200c8cfa4b1f2d4686469e9a0d2ee, we no longer have a goroutine to panic. panic should occur immediately and block the goroutine from executing the next line anyway, making the return unnecessary.

We already stopped doing return for most of these calls. It's just a couple of the really old line that didn't get cleaned up.